### PR TITLE
EKS Module Upgrade

### PR DESCRIPTION
**Pull request by vijay-stephen**
* Cloudpose version upgrade
* Removed unwanted variables anda data lookups

### CODE REVIEW

File: example/dev.auto.tfvars [hunk 1]
```diff
@@ -1,16 +1,15 @@
...
-environment                       = "dev"
-profile                           = "default"
-namespace                         = "refarchdevops"
-route_53_zone                     = "sfrefarch.com"
+environment                       = "poc"
+namespace                         = "arc"
+route_53_zone                     = "sfarcpoc.com"
 name                              = "sl-k8s"
-kubernetes_version                = "1.21"
+kubernetes_version                = "1.25"
```
Comment (mayank0202) [inline example/dev.auto.tfvars:6]: Can we try with `1.28` version? After all `1.29 `is the latest one i guess

File: example/dev.auto.tfvars [hunk 2]
```diff
@@ -32,19 +31,5 @@ map_additional_iam_roles = [
...
-  "refarchdevops-dev-publicsubnet-public-us-east-1b"
-]
-tags = {
-  Environment = "dev"
-  ENV         = "dev"
-  Project     = "sf-ref-arch"
-  Creator     = "terraform"
-}
-health_check_domains = ["healthcheck-example.sfrefarch.com"]
+vpc_name             = "arc-poc-vpc"
```
Comment (mayank0202) [inline example/dev.auto.tfvars:34]: we can fetch the vpc name from data lookups

File: example/main.tf [hunk 3]
```diff
@@ -1,6 +1,16 @@
...
+}
+
+module "tags" {
+  source      = "sourcefuse/arc-tags/aws"
+  version     = "1.2.2"
+  environment = var.environment
+  project     = "arc"
+
+  extra_tags = {
+    Repo = "github.com/sourcefuse/terraform-aws-arc-bootstrap"
```
Comment (mayank0202) [inline example/main.tf:12]: extra tags would be for eks not for bootstrap

**Review by mayank0202 — COMMENTED** [review]
Please resolve tflint issues

File: example/dev.auto.tfvars (hunk 1 above)
Comment (vijay-stephen) [reply example/dev.auto.tfvars:6]: yeah , will be testing it

File: example/dev.auto.tfvars (hunk 2 above)
Comment (vijay-stephen) [reply example/dev.auto.tfvars]: removed it from tfvars

**Review by tsaucier-sf — APPROVED** [review]

**Review by mayank0202 — APPROVED** [review]

**Pull request merged**